### PR TITLE
Adding Jitter Operation OIDs

### PR DESCRIPTION
**Pull request by naveenkaredi**
Adding Jitter Operation OIDs
 1.3.6.1.4.1.9.9.42.1.5.2.1.26 -rttMonLatestJitterOperPacketLossSD
1.3.6.1.4.1.9.9.42.1.5.2.1.27 - rttMonLatestJitterOperPacketLossDS 1.3.6.1.4.1.9.9.42.1.5.2.1.46 - rttMonLatestJitterOperAvgJitter

### CODE REVIEW

File: profiles/kentik_snmp/cisco/cisco-all-devices.yml [hunk 1]
```diff
@@ -1084,7 +1084,21 @@ metrics:
         column:
           name: rttMonEchoAdminURL 
           OID: 1.3.6.1.4.1.9.9.42.1.2.2.1.15
-
+    # A table which contains the status of latest Jitter operation
+    table:
```
Comment (thezackm) [inline profiles/kentik_snmp/cisco/cisco-all-devices.yml:1088]: this is missing the MIB designation; should look like this:

```yaml
  # A table which contains the status of latest Jitter operation
  - MIB: CISCO-RTTMON-MIB
    table:
```

**Review by thezackm — CHANGES_REQUESTED** [review]
@naveenkaredi - thank you for the submission. one small change is needed.

File: profiles/kentik_snmp/cisco/cisco-all-devices.yml (hunk 1 above)
Comment (naveenkaredi) [reply profiles/kentik_snmp/cisco/cisco-all-devices.yml:1088]: @thezackm I've added this as second table under CISCO-RTTMON-MIB, for every new table do we need to add MIB once again? 
If yes do you want me to close this PR and new one?

Comment (thezackm) [reply profiles/kentik_snmp/cisco/cisco-all-devices.yml:1088]: yes; each of these tables are part of a distinct MIB "object"; no need for a new PR; just make the edit on your branch and it should "automagically" show up here in this PR

Comment (naveenkaredi) [reply profiles/kentik_snmp/cisco/cisco-all-devices.yml]: @thezackm Thank you Sir, ([Added MIB Object](https://github.com/kentik/snmp-profiles/pull/735/commits/755096df308fa929538b26bbd9631d1fa8b6a0e2)) can you please check and approve?

**Review by thezackm — APPROVED** [review]
thanks for the submission; this should be available on the latest docker image in about 60 minutes

**Comment by naveenkaredi** [comment]
Thanks Zack!

**Pull request merged**